app/controllers/box_request_triage_controller.rb

```diff
@@ -0,0 +1,38 @@
+class BoxRequestTriageController < ApplicationController
+
+  def create
+    if request_params.empty?
+      head :unprocessable_entity
+      return
+    end
+
+    @payload = JSON.parse(request_params[:boxRequest]).with_indifferent_access
+
+    requester = Requester.new
+
+    #TODO some of the existing fields (is_interested_in_counseling_services, is_interested_in_health_services, question_re_affect, question_re_current_situation, question_re_if_not_self_completed, question_re_referral_source) are not in our db for the time being
+    [:first_name,
+     :last_name,
+     :email,
+     :street_address,
+     :city,
+     :state,
+     :zip,
+     :ok_to_email,
```

Comment:
we had originally anticipated the following additional fields on Requester:
- phone
- county
- underage

Reply:
were these removed from (or not added to) the form and/or database?

Reply:
for underage, that can become a tag on the BoxRequest, but think we should still add it to the Requester table (even tho that data will become stale, i think we would want to retain that they were underage when we added them to the system)